ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/WhisperClient.cs

```diff
@@ -55,21 +55,10 @@ public async Task FormAsync(WhisperForm message, CancellationToken cancellationT
 
         private static WhisperMeta GenerateMeta(WhisperConfig config)
         {
-            var styleBackgroundColor =
```

Comment:
We can also now drop the default values (L11-13)  and the coercion function (L76-79)

Reply:
Thanks for pointing that out. They have been removed.